src/AutoViz.jl

```diff
@@ -16,7 +16,7 @@ const DEFAULT_CANVAS_WIDTH = 1000
 const DEFAULT_CANVAS_HEIGHT = 600
 export DEFAULT_CANVAS_WIDTH, DEFAULT_CANVAS_HEIGHT
 
-global rendermode = :fancy
+global rendermode = :basic # :fancy
 
```

Comment:
I'd like to propose making `:basic` the default render mode rather than `:fancy`, this seems to be a more sensible choice.
Also, currently it seems that `AutoViz.rendermode = :fancy` does not work, one needs to run `@eval(AutoViz, rendermode=:fancy)` in order to change the rendermode. Maybe we can make this more convenient using a function `rendermode(:fancy)` that sets the corresponding variables.

Reply:
a function sounds better yes, but why not having the nice cars by default?

Reply:
My thoughts were the following (in no particular order of importance)
 - "basic" cars provide better "introspection", i.e. the outlines of the vehicle are clearer, the velocity is shown as an arrow and helps with understanding what's going on, no car features to distract from the actual simulation
 - in my perception, "default" is almost a synonym of "basic", i.e. simple, essential. Bells and whistles or "fancy" looks are "extras" (at least for me)
- simple squares should be faster to render since they don't require any SVG reading and are less complex - I did not actually test this but it subjectively felt like it may be the case

I think my personal use case would even be to use simple rectangles for development and testing, and use "fancy" rendering for the final run to make a cool visualization once everything is working.

Reply:
I see, that makes sense, we want to showcase the fancy rendering in the README and docs though.

Reply:
> I see, that makes sense, we want to showcase the fancy rendering in the README and docs though.

It's there in the docs in "AutoViz Basics"